packages/itwinui-css/src/utils/icon/icon.scss

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../../style/index';
+
+/// @arg $size Can be one of: s, m, l, xl, 2xl, 3xl
+@mixin iui-icon-style($size: 'm') {
+  display: flex;
+  flex-shrink: 0;
+  @if $size == 's' {
+    width: $iui-icon-small;
+    height: $iui-icon-small;
+  }
+  @if $size == 'm' {
+    width: $iui-icon-default;
+    height: $iui-icon-default;
+  }
+  @if $size == 'l' {
+    width: $iui-icon-large;
+    height: $iui-icon-large;
+  }
+  @if $size == 'xl' {
+    width: $iui-icon-xl;
+    height: $iui-icon-xl;
+  }
+  @if $size == '2xl' {
+    width: $iui-icon-2xl;
+    height: $iui-icon-2xl;
+  }
+  @if $size == '3xl' {
+    width: $iui-icon-3xl;
+    height: $iui-icon-3xl;
+  }
+}
+
+@mixin iui-icon-color {
```

Comment:
Suggestion to replace `@mixin iui-icon-color` and `@mixin iui-icon-color-status` with this:
```scss
/// @arg $color Can be one of: informational, positive, warning, negative, or left blank for default color
@mixin iui-icon-color($color) {
  @if $color == 'informational' {
    fill: var(--iui-icon-color-primary);
  }
  @if $color == 'positive' {
    fill: var(--iui-icon-color-positive);
  }
  @if $color == 'warning' {
    fill: var(--iui-icon-color-warning);
  }
  @if $color == 'negative' {
    fill: var(--iui-icon-color-negative);
  }
  @else {
    fill: var(--iui-icon-color);
  }
}
```

Which can be used in `classes.scss` like this:
```scss
.iui-icon-color {
  @include iui-icon-color;
}
.iui-icon-color-informational {
  @include iui-icon-color(informational);
}
.iui-icon-color-positive {
  @include iui-icon-color(positive);
}
.iui-icon-color-warning {
  @include iui-icon-color(warning);
}
.iui-icon-color-negative {
  @include iui-icon-color(negative);
}
```

That would keep our classes a little bit cleaner:
```diff
- class="iui-icon-style-m iui-icon-color-status iui-informational"
+ class="iui-icon-style-m iui-icon-color-informational"
```

Reply:
How would this work for the places like this?
![image](https://user-images.githubusercontent.com/37936169/181581960-7c222bfe-e303-4f6a-adca-647bc53301f4.png)

Reply:
Leah and I just had a call about this, I can think of 2 or 3 different ways we could handle this & would like to make a team decision in our call tomorrow.

One option is to place this in place of all instances of `@include iui-icon-color-status`:
```scss
.iui-tile-status-icon {
  @include iui-icon-style('m');
  @include iui-icon-color;
  margin-right: $iui-s;

  &.iui-icon-color-informational {
    @include iui-icon-color(informational);
  }
  &.iui-icon-color-positive {
    @include iui-icon-color(positive);
  }
  &.iui-icon-color-warning {
    @include iui-icon-color(warning);
  }
  &.iui-icon-color-negative {
    @include iui-icon-color(negative);
  }
}
```

Another option is we have a mixin to handle all class modifiers (example: `.iui-tile-status-icon .iui-informational`) that is _not_ used in `classes.scss`.  I think this is my least favorite option but it also introduces the least amount of changes for pre-existing components like tile & expandable block.

The last option I can think of is to use data attributes for icon color.  Because the data attribute is specific (`data-iui-icon-color` VS `data-iui-color`) it doesn't need to be nested under a parent class.  This would remove the need for `@include iui-icon-color-status` inside expandable block & tile.
```scss
[data-iui-icon-color] {
  @include iui-icon-color;
}

[data-iui-icon-color='informational'] {
  @include iui-icon-color(informational);
}

[data-iui-icon-color='positive'] {
  @include iui-icon-color(positive);
}

[data-iui-icon-color='warning'] {
  @include iui-icon-color(warning);
}

[data-iui-icon-color='negative'] {
  @include iui-icon-color(negative);
}
```

Reply:
I think if we use data attributes for color, we should use them for icon size also.

Reply:
I would disagree with having such mixins. More code - same result because you need to explicitly enumerate all status colors where with current mixin you just include it and have all the colors available.